sdk/go/azure/webpubsub/customCertificate.go

```diff
@@ -14,6 +14,123 @@ import (
 
 // Manages an Azure Web PubSub Custom Certificate.
 //
+// ## Example Usage
+//
+// ```go
+// package main
+//
+// import (
+//
+//	"github.com/pulumi/pulumi-azure/sdk/v5/go/azure/core"
+//	"github.com/pulumi/pulumi-azure/sdk/v5/go/azure/keyvault"
+//	"github.com/pulumi/pulumi-azure/sdk/v5/go/azure/webpubsub"
+//	"github.com/pulumi/pulumi-azurerm/sdk/v1/go/azurerm"
+//	"github.com/pulumi/pulumi-std/sdk/go/std"
+//	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+//
+// )
+//
+//	func main() {
+//		pulumi.Run(func(ctx *pulumi.Context) error {
+//			current, err := core.GetClientConfig(ctx, nil, nil)
+//			if err != nil {
+//				return err
+//			}
+//			example, err := core.NewResourceGroup(ctx, "example", &core.ResourceGroupArgs{
+//				Name:     pulumi.String("example-resources"),
+//				Location: pulumi.String("West Europe"),
+//			})
+//			if err != nil {
+//				return err
+//			}
+//			exampleWebPubsubService, err := index.NewWebPubsubService(ctx, "example", &index.WebPubsubServiceArgs{
+//				Name:              "example-webpubsub",
+//				Location:          testAzurermResourceGroup.Location,
+//				ResourceGroupName: testAzurermResourceGroup.Name,
+//				Sku: []map[string]interface{}{
+//					map[string]interface{}{
+//						"name":     "Premium_P1",
+//						"capacity": 1,
+//					},
+//				},
+//				Identity: []map[string]interface{}{
+//					map[string]interface{}{
+//						"type": "SystemAssigned",
+//					},
+//				},
+//			})
+//			if err != nil {
+//				return err
+//			}
+//			exampleKeyVault, err := keyvault.NewKeyVault(ctx, "example", &keyvault.KeyVaultArgs{
+//				Name:              pulumi.String("examplekeyvault"),
+//				Location:          example.Location,
+//				ResourceGroupName: example.Name,
+//				TenantId:          *pulumi.String(current.TenantId),
```

Comment:
You can't dereference a `pulumi.String`.

It looks like we do that a lot in this example.

Reply:
https://github.com/pulumi/pulumi/issues/15524

Reply:
Is this blocking further converter rollout, or just something that we should fix as soon as we can?

Reply:
That's a good question Justin, we implicitly decided to go ahead with the rollout in spite of this issue though it seems fairly prevalent. It should be after all easy to fixup manually if someone is copying the example. Do you think we should revert out and hold?

Reply:
Let's not revert. We can fix forward.